ParsePush/build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'com.android.library'
+
+group = 'com.parse'
+version = '1.6.1'
```

Comment:
If you hardcode the version number, I do not think you still need `gradle.properties`.
The two versions are not match, which one is the right one?